e107_admin/header.php

```diff
@@ -83,11 +83,11 @@ function loadJSAddons()
 	e107::js('footer', '{e_WEB}js/core/admin.jquery.js', 'jquery', 5); // Load all default functions.
 	e107::js('footer', '{e_WEB}js/core/all.jquery.js', 'jquery', 5); // Load all default functions.
 
-	$plUpload = '{e_WEB}js/plupload/i18n/' . e_LAN . '.js';
+	$plUpload = 'plupload/i18n/' . e_LAN . '.js';
 
 	if(e_LAN != 'en' && file_exists(e_WEB_JS . $plUpload))
 	{
-		e107::js('footer', $plUpload, 'jquery', 5);
+		e107::js('core', $plUpload, 'jquery', 5);
 	}
 }
 
```

Comment:
It would be better in the footer if possible. Does this work?

    $plUpload = 'plupload/i18n/' . e_LAN . '.js';

    if(e_LAN != 'en' && file_exists(e_WEB_JS . $plUpload))
 	{
         e107::js('footer', e_WEB_JS. $plUpload, 'jquery', 5);
 	}

Reply:
Great! Work this!